### ISSUE TITLE
gitbeaker repository-files show fails

### ISSUE THREAD

**Issue by detj** [jdalrymple/gitbeaker#833]
`gitbeaker repository-files show` CLI command fails with 404 status.

**Description**

Running the following command on the [`gitlab-foss`](https://gitlab.com/gitlab-org/gitlab-foss) repository fails.

```bash
$ gitbeaker repository-files show --project-id=13083 --ref=master --file-path=Gemfile
```

**Steps to reproduce**
1. Install `gitbeaker` CLI

```bash
$ npm install -g @gitbeaker/cli
```

2. Run the following command

```bash
$ gitbeaker repository-files show --project-id=13083 --ref=master --file-path=Gemfile
```

**Expected behaviour**
Should output the correct JSON payload for the "Gemfile" file path.

```json
{
  "file_name": "Gemfile",
  "file_path": "Gemfile",
  "size": 11869,
  "encoding": "base64",
  "content_sha256": "c896e3d29930c91006dc6b728877b4bf8b858c9f0208aea47d884f29bc039906",
  "ref": "master",
  "blob_id": "9c8c5e8b30d7d6f10c63331645c74deae7fc868d",
  "commit_id": "0e54270ecfcd621d6bba749dd484e4abac83a6ff",
  ...
}
```

To verify, you can generate the response using curl.

```bash
$ curl -s https://gitlab.com/api/v4/projects/13083/repository/files/Gemfile?ref=master
```

**Actual behaviour**
Fails with following error

```
⠧ Calling GitlabHTTPError: Response code 404 (Not Found)
    at onResponse (/Users/xxxx/n/lib/node_modules/@gitbeaker/cli/node_modules/got/dist/source/as-promise/index.js:124:28)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: undefined,
  timings: {
    start: 1590068259656,
    socket: 1590068259658,
    lookup: 1590068259784,
    connect: 1590068259830,
    secureConnect: 1590068259883,
    upload: 1590068259883,
    response: 1590068260272,
    end: 1590068260277,
    error: undefined,
    abort: undefined,
    phases: {
      wait: 2,
      dns: 126,
      tcp: 46,
      tls: 53,
      request: 0,
      firstByte: 389,
      download: 5,
      total: 621
    }
  },
  description: undefined
}
✖ Calling Gitlab
```

**Possible fixes**
My initial guess is the request being sent by gitbeaker is incorrect. Not debugged further.

**Comment by detj** [comment]
This works though.

```js
const { RepositoryFiles } = require("@gitbeaker/node");

const api = new RepositoryFiles();

api
  .show("13083", "Gemfile", "master")
  .then((res) => {
    console.log(res);
  })
  .catch((e) => {
    console.error(e);
  });
```

Returns correct response. So, something must be wrong with the CLI? 

Also wondering how many other resources maybe affected. 🤔

**Comment by jdalrymple** [comment]
Hmm i wonder why. Ill give it a look this week when i get a spare moment!

**Comment by jdalrymple** [comment]
Founnnnnd the problem. Fixing it rn!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`21.5.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/21.5.0) :rocket:
